src/components/modal/Modal.js

```diff
@@ -0,0 +1,29 @@
+import ReactModal from 'react-modal';
```

Comment:
`Uncaught Error: Cannot find module 'react-modal'`
I don't see `react-modal` in your [dependencies](https://github.com/OlgaLyakhvatskaya/homework/blob/saga/package.json)